CHANGELOG.rst

```diff
@@ -1,13 +1,17 @@
 Changelog
 =========
 
-UNRELEASED
-~~~~~~~~~~
+1.2.1 (2020-05-19)
```

Comment:
I think we should go to 1.3.0. WDYT?

Reply:
Hesitated on this point, let's go with  1.3.0! I'll send in the changes

Reply:
Alright 1.3.0 it has become